randovania/games/dread/json_data/header.txt

```diff
@@ -111,31 +111,222 @@ Templates
+          Power Bombs ≥ 2 and Knowledge (Beginner) and Lay Power Bomb
+          All of the following:
+              Any of the following:
+                  # Methods of dodging
+                  Flash Shift or Combat (Beginner)
+              Any of the following:
+                  # Methods of fighting
+                  Shoot Plasma Beam
+                  All of the following:
+                      # 39 Super Missiles are required, 1.5x leeway for trickless
+                      Super Missile and Shoot Missile
+                      Any of the following:
+                          Missiles ≥ 59
+                          Missiles ≥ 39 and Combat (Beginner)
+                  All of the following:
+                      # 50 Ice Missiles required, 1.5x leeway for trickless
+                      Shoot Ice Missile
+                      Any of the following:
+                          Missiles ≥ 75
+                          Missiles ≥ 50 and Combat (Beginner)
+                  All of the following:
+                      # 40 Storm Missiles required, 1.5x leeway for trickless
+                      Storm Missile and Shoot Missile
+                      Any of the following:
+                          Missiles ≥ 60
+                          Missiles ≥ 140 and Combat (Beginner)
+                  Combat (Advanced) and Shoot Beam
+                  Combat (Intermediate) and Shoot Wide Beam
+
+* Fight Red Chozo-X:
+      Any of the following:
+          Power Bombs ≥ 2 and Knowledge (Beginner) and Lay Power Bomb
+          All of the following:
+              Any of the following:
+                  # Methods of dodging
+                  Flash Shift or Combat (Beginner)
+              Any of the following:
+                  # Methods of fighting
+                  All of the following:
+                      # 54 Super Missiles are required, 1.5x leeway for trickless
+                      Shoot Super Missile
+                      Any of the following:
+                          Missiles ≥ 81
+                          Missiles ≥ 54 and Combat (Beginner)
+                  All of the following:
+                      # 40 Ice Missiles are required, 1.5x leeway for trickless
+                      Shoot Ice Missile
+                      Any of the following:
+                          Missiles ≥ 60
+                          Missiles ≥ 40 and Combat (Beginner)
+                  All of the following:
+                      # 50 Storm Missiles required, 1.5x leeway for trickless
+                      Storm Missile and Shoot Missile
+                      Any of the following:
+                          Missiles ≥ 75
+                          Missiles ≥ 50 and Combat (Beginner)
+                  Combat (Intermediate) and Shoot Wide Beam
+                  Combat (Advanced) and Shoot Beam
+
+* Fight Gold Chozo-X:
+      Any of the following:
+          Power Bombs ≥ 2 and Knowledge (Beginner) and Lay Power Bomb
```

Comment:
Gold chozo-x is NOT vulnerable to power bombs

Reply:
Oh, oops. i had copied this from the red chozo-x template initially